util/parallel.cpp

```diff
@@ -235,6 +236,7 @@ bool parallel::emitOutput() {
       out_file << line << '\n';
     }
   }
+  out_file.flush();
```

Comment:
These flushes were necessary to avoid a buggy behavior that is exactly #720 when batching is enabled. When batching is disabled, this bug didn't appear again.
I investigated quite amount of time for this but I couldn't find why... adding flush calls here worked and passed Dongjoo's sanity checks for the printed logs.

Reply:
These extra flushes were still necessary. :(

Reply:
I've fixed the problem here: https://github.com/AliveToolkit/alive2/commit/0f41f08816619250c96934bb9169f8bb5f500745

The flush was in the wrong place.